crates/ruff_linter/src/fix/edits.rs

```diff
@@ -144,6 +144,26 @@ pub(crate) fn make_redundant_alias<'a>(
         .collect()
 }
 
+/// Fix to add the specified imports to the `__all__` export list.
+pub(crate) fn add_to_dunder_all<'a>(
+    names: impl Iterator<Item = &'a str>,
+    dunder_all: &ExprList,
+) -> Vec<Edit> {
+    let insertion_point = dunder_all
+        .elts
+        .last()
+        .map_or(dunder_all.range.end() - TextSize::from(1), |expr| {
+            expr.range().end()
+        });
+    names
+        .enumerate()
+        .map(|(offset, name)| match dunder_all.elts.len() + offset {
+            0 => Edit::insertion(format!("{name:?}"), insertion_point),
+            _ => Edit::insertion(format!(", {name:?}"), insertion_point),
```

Comment:
I believe this is going to mean that the added strings will always use double quotes, but that might look odd if all the other strings in the `__all__` sequence use single quotes. Consider using the `ruff_python_codegen::Stylist::quote()` method to determine the standard quoting style used by the source code in question, e.g.

https://github.com/astral-sh/ruff/blob/6774f27f4b96725084b04711aa5aec2c6ac2a597/crates/ruff_linter/src/rules/flake8_type_checking/helpers.rs#L278-L284

You can get a `Stylist` instance by calling `checker.stylist()` in a Ruff rule that has access to a `Checker` instance.

Reply:
Nice find!

Reply:
Great! I was wondering how to do this properly. Willfix